### ISSUE TITLE
Feature Request: Ability to Promote an Input node to Be visible/accessible from top level of graph

### ISSUE THREAD

**Issue by eric-schleicher** [vizorvr/vizor#752]
I would be very useful to be able to specify (for an input node) that it should be available at the top of the graph (or in some alternate UI the user has quick and easy access to)

very helpful for building and debugging graphs, cuts down on the amount of network traversal the designer has to do to get to commonly changes inputs.

The ability to specify such a value for an input can also be used to control what degrees of freedom  (options) would be exposed to the user in a non-design/presentation environment.

**Comment by fthr** [comment]
Yeah, this would be extremely useful. We've discussed various UI ideas for it. One of them is some kind of "presentation mode" or "control layer". User could define patches that are shown on this layer, and that could be presented to end users as the way to control predefined parameters.

A step further would be an easy way to export these parameters to the published player, something like dat.gui  https://code.google.com/p/dat-gui/
